src/Platform/Microsoft.Testing.Platform/OutputDevice/TargetFrameworkParser.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Globalization;
+
+namespace Microsoft.Testing.Platform.OutputDevice;
+
+internal static class TargetFrameworkParser
+{
+    internal /* for testing */ static string? GetShortTargetFramework(string frameworkDescription)
+    {
+        // https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation.frameworkdescription?view=net-8.0
```

Comment:
Nit: let's be cultural friendly :)

```suggestion
        // https://learn.microsoft.com/dotnet/api/system.runtime.interopservices.runtimeinformation.frameworkdescription
```